### PR TITLE
First stab issue templates

### PR DESCRIPTION
**Pull request by justb4**
Three issue type templates.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG] in GeoHealthCheck (replace with your title}"
```
Comment (tomkralidis) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Do we need `[BUG]` in the title?  We usually label as a bug as part triaging?

File: .github/ISSUE_TEMPLATE/question.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+name: Question
+about: I have a question
+title: "[QUESTION] I have a question!"
```
Comment (tomkralidis) [inline .github/ISSUE_TEMPLATE/question.md]: Same here, do we need `[QUESTION]` in the title?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (justb4) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Ok, fixed.

File: .github/ISSUE_TEMPLATE/question.md (hunk 2 above)
Comment (justb4) [reply .github/ISSUE_TEMPLATE/question.md]: Ok, fixed.

**Pull request merged**